### ISSUE TITLE
Better docs request: A HOWTO guide for users upgrading between 1.0.x and 1.1.x

### ISSUE THREAD

**Issue by DrupaListo-com** [borgbackup/borg#3320]
As I recently upgraded from 1.0.10 to 1.1.2 I noticed that the changelog is GOOD, but it can be better.
I noticed that there is even a summary section in the changelog.

But optimally/ultimately I'd like us to add a new page in the borg docs: "howto upgrade from 1.0 to 1.1 - list of things to check" - because even the current changelog is not summarized enough and some things are not obviuous and clear enough.

Possible contents of that page:
- notable borg options changes and new commands added (already covered by changelog, maybe can be written better)
- a specific answer to the question - "should I do anything special on upgrade and if yes - what?"
  - YES the changelog mentions that there is no need to run "borg upgrade", but due to some notes about better data structures that are backwards compatible etc, I was left in doubt about whether my 1.0 repo will cause borg 1.1 to work in some inferior backwards compatible mode or there will be no difference / penalty to have a 1.0 repo in a borg 1.1 world...

Sorry for a long post. I think couple of re-wordings and sentences added to the changelog could suffice.
It would be even nicer if we had a dedicated and simple upgrade howto.

What does the community think?

**Comment by DrupaListo-com** [comment]
oh and we forgot to mention in a notable way in the changelog that 1st run of 1.1 on a 1.0 repo will take a VERY long time (15min usually, 1h 30min on 1st 1.1 run) - because the whole cache is re-checked/re-built.

Another thing that needs MORE attention is the new --files-cache option
Near the changelog start it is barely mentioned - I quote:

"borg create: automatic compression mode (-C auto,zlib/lz4/...), faster handling of many files, experimental support for inclusion patterns (--patterns, --patterns-from), files cache mode control (mtime/ctime)"

**Comment by enkore** [comment]
I think the change the caused [that 1st run of 1.1 on a 1.0 repo will take a VERY long time] was kinda last minute which is probably why it isn't even mentioned in the summary.

If you want you can PR against https://github.com/borgbackup/borgbackup.github.io/blob/source/releases/borg-1.1.rst

**Comment by DrupaListo-com** [comment]
update: I PR-ed today against that file in https://github.com/borgbackup/borgbackup.github.io/pull/19
I suppose this issue's place is in the borgbackup.github.io repo's issue section, but nevermind.

Finally I now saw that the Long Changelog (for 1.1.2 in the Compatibility notes: section) HAS the same info I proposed in the PR above for the Short changelog, yay!

This issue can be closed now I suppose as the PR in borgbackup.github.io will be enough to not lose track of things.
